static/app/components/sidebar/index.tsx

```diff
@@ -492,10 +497,12 @@ function Sidebar() {
   return (
     <SidebarWrapper aria-label={t('Primary Navigation')} collapsed={collapsed}>
       <SidebarSectionGroupPrimary>
-        <DropdownSidebarSection isSuperuser={hasSuperuserSession}>
+        <DropdownSidebarSection isSuperuser={hasSuperuserSession()}>
```

Comment:
since we only utilize the hook in here, could we isolate calling it outside of hasSuperuserSession? or make another variable like `showRedSidebar`